src/pages/payments/widgets/my-cards/PaymentCardList.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="grid md:grid-cols-2 grid-cols-1 gap-4">
+    <VaSkeletonGroup
+      v-if="loading"
+      class="min-h-[114px] p-4 rounded-lg border border-dashed border-backgroundBorder flex flex-row items-center gap-6"
+    >
+      <div class="flex flex-col gap-2 flex-grow">
+        <VaSkeleton class variant="text" width="10rem" height="1.5rem" />
+        <div class="flex gap-4">
+          <VaSkeleton variant="rounded" width="5rem" height="3rem" />
+          <VaSkeleton variant="text" :lines="2" />
+        </div>
+      </div>
+    </VaSkeletonGroup>
+    <template v-else>
+      <CardListItem
+        v-for="paymentCard in list"
+        :key="paymentCard.id"
+        :card="paymentCard"
+        @remove="remove(paymentCard)"
+        @edit="cardToEdit = paymentCard"
+      />
+      <div
+        class="sm:h-[114px] p-4 bg-primary bg-opacity-5 rounded-lg border border-dashed border-primary flex flex-col sm:flex-row items-start sm:items-center gap-4"
+      >
+        <div class="flex flex-col gap-2 flex-grow">
+          <div class="text-lg font-bold leading-relaxed">Important note</div>
+          <div class="text-secondary text-sm leading-tight">
+            Please carefully read
+            <span class="underline">Product Terms</span> before adding your new payment card
+          </div>
+        </div>
+        <va-button class="flex-none w-full sm:w-auto" @click="showCreate = true">Add card</va-button>
+      </div>
+    </template>
+  </div>
+  <PaymentCardCreateModal v-if="showCreate" @close="showCreate = false" />
+  <PaymentCardUpdateModal v-if="cardToEdit" :payment-card="cardToEdit" @close="cardToEdit = ''" />
```

Comment:
```suggestion
  <PaymentCardUpdateModal v-if="cardToEdit" :payment-card="cardToEdit" @close="cardToEdit = undefined" />
```
![image](https://github.com/epicmaxco/vuestic-admin/assets/23530004/54602d8c-2034-430f-9af9-a2eb68b233ca)

Same in billing address